tests/src/Kernel/ViewsLinkFieldsTest.php

```diff
@@ -0,0 +1,149 @@
+use Drupal\views\Views;
+
+/**
+ * Tests link fields to preview and edit medias in views.
+ */
+class ViewsLinkFieldsTest extends KernelTestBase {
+
+    use UserCreationTrait;
+    use MediaCreationTrait;
+    use MediaTypeCreationTrait;
+
+    /**
+     * Media owned by current user.
+     *
+     * @var \Drupal\media\MediaInterface;
+     */
+    protected $ownMedia = NULL;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static $modules = [
+        'collabora_online',
+        'collabora_online_test',
+        'field',
+        'file',
+        'image',
+        'media',
+        'system',
+        'user',
+        'views',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp(): void {
+        parent::setUp();
+
+        $this->installEntitySchema('file');
+        $this->installEntitySchema('media');
+        $this->installEntitySchema('user');
+        $this->installConfig(['user', 'views', 'collabora_online_test']);
+        $this->installSchema('file', ['file_usage']);
+        // Install user module avoid user 1 permissions bypass.
+        \Drupal::moduleHandler()->loadInclude('user', 'install');
+        user_install();
+
+        // Create two medias to check access with different scopes, 'any' and 'own'.
+        $this->createMediaEntity('document');
+        $this->ownMedia = $this->createMediaEntity('document');;
+    }
+
+    /**
+     * Tests link fields.
+     */
+    public function testLinks(): void {
+        // User without permissions can't see links.
+        $this->doTestLinks([
+            'preview' => [FALSE, FALSE],
+            'edit' => [FALSE, FALSE],
+        ],
+        $this->createUser([]));
+        // User with 'Preview' permission can see preview link.
+        $this->doTestLinks([
+            'preview' => [TRUE, TRUE],
+            'edit' => [FALSE, FALSE],
+        ],
+        $this->createUser([
+        'preview document in collabora'
+        ]));
+        // User with 'Edit any' permission can see edit link.
+        $this->doTestLinks([
+            'preview' => [FALSE, FALSE],
+            'edit' => [TRUE, TRUE],
+        ],
+        $this->createUser([
+            'edit any document in collabora'
+        ]));
+        // User with 'Edit own' permission can see edit link for entities they own.
+        $this->doTestLinks([
+            'preview' => [FALSE, FALSE],
+            'edit' => [FALSE, TRUE],
+        ],
+    $this->createUser([
+            'edit own document in collabora'
+        ]));
+    }
+
+    /**
+     * Tests that links behave as expected.
+     *
+     * @param [] $expected_results
+     *   An associative array of expected results keyed by operation.
+     * @param \Drupal\Core\Session\AccountInterface $account
+     *   The user account to be used to run the test;
+     */
+    protected function doTestLinks(array $expected_results, AccountInterface $account) {
+        $this->setCurrentUser(account: $account);
```

Comment:
Named parameter syntax is not needed here.
```suggestion
        $this->setCurrentUser($account);
```
I can think of a few reasons to use named parameters in calls, but none of them apply here:
- to skip other optional parameters. E.g. `new \Exception($msg, prev: $e)`.
- to add clarity, if the purpose of a parameter in a long parameter list is not obvious. Especially if the function has many parameters, or if it is a boolean parameter where it is easy to confuse true and false. (with PhpStorm the param names are displayed in call, so this becomes less important).
- to align with other code elsewhere.

Reply:
Unintended, IDE shows what is the name of the paremeters and allows to add them as named paremeters by clicking them, most probably a missed click.

Removing it.